cypress/integration/oops_option.js

```diff
@@ -2,7 +2,7 @@
 describe('Always show oops setting', function() {
   it('shows oops message', function() {
     cy.visit('localhost:8080/test.html')
-    cy.get('#non_cree_hello').click()
+    cy.get('#Chinese').click()
```

Comment:
We used to produce "Oops...translation not found" for _hello_
Now click-in-text translates English too.